src/canillita_newsflash.erl

```diff
@@ -0,0 +1,139 @@
+%%% @doc NewsFlash Model
+-module(canillita_newsflash).
+
+-behaviour(sumo_doc).
+-behaviour(sumo_rest_doc).
+
+-type id() :: binary().
+-type newspaper_name() :: binary().
```

Comment:
I would even just remove the type altogether and directly use the one in canillita_newspaper everywhere